ultralytics/engine/predictor.py

```diff
@@ -300,7 +300,7 @@ def setup_model(self, model, verbose=True):
             dnn=self.args.dnn,
             data=self.args.data,
             fp16=self.args.half,
-            batch=self.args.batch,
+            batch=int(np.ceil(self.args.batch).clip(1)),
```

Comment:
@Burhan-Q we don't want to constrain/verify these values I think in any other place than check_cfg(), and we should only do this once rather than repeat the implementation in multiple places.